dask/dataframe/tests/test_dataframe.py

```diff
@@ -186,28 +186,57 @@ def test_split_apply_combine_on_series():
         assert eq(d.groupby(ddkey).a.min(), full.groupby(pdkey).a.min())
         assert eq(d.groupby(ddkey).a.max(), full.groupby(pdkey).a.max())
         assert eq(d.groupby(ddkey).a.count(), full.groupby(pdkey).a.count())
+        assert eq(d.groupby(ddkey).a.mean(), full.groupby(pdkey).a.mean())
+        assert eq(d.groupby(ddkey).a.nunique(), full.groupby(pdkey).a.nunique())
+
+        assert eq(d.groupby(ddkey).sum(), full.groupby(pdkey).sum())
+        assert eq(d.groupby(ddkey).min(), full.groupby(pdkey).min())
+        assert eq(d.groupby(ddkey).max(), full.groupby(pdkey).max())
+        assert eq(d.groupby(ddkey).count(), full.groupby(pdkey).count())
+        assert eq(d.groupby(ddkey).mean(), full.groupby(pdkey).mean())
```

Comment:
Oh, neat.  I had no idea that this was valid Pandas code.  It's nice how little work there was to make this work.